packages/flutter_test/lib/src/controller.dart

```diff
@@ -445,17 +441,25 @@ abstract class WidgetController {
     return result;
   }
 
-  /// Begins a gesture at a particular point, and returns the
-  /// [TestGesture] object which you can use to continue the gesture.
-  Future<TestGesture> startGesture(Offset downLocation, { int pointer }) {
-    return TestGesture.down(
-      downLocation,
-      pointer: pointer ?? _getNextPointer(),
+  /// Creates a gesture and returns the [TestGesture] object which you can use
+  /// to continue the gesture.
```

Comment:
nit: would probably be good to call out in the doc the difference between create and start gesture.

Reply:
Done.